x509/src/certificate.rs

```diff
@@ -0,0 +1,243 @@
+//! Certificate [`Certificate`] and TBSCertificate [`TBSCertificate`] as defined in RFC 5280
+
+use crate::{Name, Validity};
+use der::asn1::{BitString, ContextSpecific, ObjectIdentifier, UIntBytes};
+use der::{Sequence, TagMode, TagNumber};
+use spki::{AlgorithmIdentifier, SubjectPublicKeyInfo};
+
+/// returns false in support of integer DEFAULT fields set to 0
+pub fn default_zero_u8() -> u8 {
+    0
+}
+
+/// returns false in support of boolean DEFAULT fields
+pub fn default_false() -> bool {
+    false
+}
+
+/// returns false in support of integer DEFAULT fields set to 0
+pub fn default_zero() -> u32 {
+    0
+}
```

Comment:
These all make me think there should be some shorthand when using the proc macro attributes for using `Default` as the default in these cases, possibly `default = true`.

Reply:
There will be more complicated defaults coming soon (i.e AlgorithmIdentifiers) so this approach would likely need to remain but could be augmented with support for simpler cases.